app/javascript/components/Results/ResultGrid/ResultGrid.tsx

```diff
@@ -43,8 +44,7 @@ export const ResultGrid = ({ vertical, result }: ResultProps) => {
             <a href={result.url} className='result-title-link'>
               <h2 className='result-title-label'>
                 {parse(result.title)} 
-                {/* ToDo: This need to be dynamic */}
-                {/* <span className='filetype-label'>PDF</span> */}
+                <span className='filetype-label'>{result.fileType}</span>
```

Comment:
Use below so that we only display if fileType is available
`{result.fileType && <span className='filetype-label'>{result.fileType}</span>}`